### ISSUE TITLE
Differences im results between version 4.27.2 and 4.27.3

### ISSUE THREAD

**Issue by msmolka** [mikefarah/yq#1333]
**Describe the bug**

Note that any how to questions should be posted in the discussion board and not raised as an issue.

Version of yq: 4.27.3
Operating system:linux/windows/....
Installed via: exe from GitHub

I have following use case

https://stackoverflow.com/questions/72810898/deep-merge-two-yaml-files-and-keep-only-new-fields

The accepted answer worked in version 4.27.2, however after moving to 4.27.3, yq is no longer producing expected result.

**Comment by mikefarah** [comment]
Yeah this looks like a bug :( 
It seems to be an issue with the `*=n` operator, for now, you can do this workaround:

```
yq 'with(.root[]; . = . *n load("update.yaml").defaults)' master.yaml
``

**Comment by mikefarah** [comment]
Fixed in 4.27.5

**Comment by msmolka** [comment]
Thank you for workaround. This issue hit with deployment on Azure Pipelines. I suddenly could not deploy anything. Even older releases, because image has only new version.
